PyNvCodec/src/PyFFMpegDecoder.cpp

```diff
@@ -123,7 +123,7 @@ py::array_t<MotionVector> PyFfmpegDecoder::GetMotionVectors()
   size /= sizeof(*ptr);
 
   if (ptr && size) {
-    py::array_t<MotionVector> mv({size});
+    py::array_t<MotionVector> mv({static_cast<ssize_t>(size)});
```

Comment:
Please remove this line as it's not relevant to commit main purpose.
If you've found redundant or unnecessary lines of code, please make it a separate PR.

Reply:
Done